docs/source/getting-started.md

```diff
@@ -39,7 +39,7 @@ Exporting is the process of taking a PyTorch model and converting it to the .pte
 ### Selecting a Backend
 ExecuTorch provides hardware acceleration for a wide variety of hardware. The most commonly used backends are XNNPACK, for Arm and x86 CPU, Core ML (for iOS), Vulkan (for Android GPUs), and Qualcomm (for Qualcomm-powered Android phones).
 
-For mobile use cases, consider using XNNPACK for Android and Core ML or XNNPACK for iOS as a first step. See [Delegates](/TODO.md) for a description of available backends.
+For mobile use cases, consider using XNNPACK for Android and Core ML or XNNPACK for iOS as a first step. See [Hardware Backends](using-executorch-export.md#hardware-backends) for more information.
```

Comment:
Why not link to the dedicated page: 

https://docs-preview.pytorch.org/pytorch/executorch/8618/backends-overview.html

Reply:
Good catch. This change predated that file's existence - I've updated it to point to that file.